app/controllers/api/v1/cinema_halls_controller.rb

```diff
@@ -7,16 +7,16 @@ def index
   end
 
   def show
-    @cinema_hall = CinemaHalls::UseCases::Find.new.call(id: params[:id])
+    @cinema_hall = CinemaHalls::UseCases::Show.new.call(id: params[:id])
```

Comment:
You don't need that use_case, you can use repository class: `CinemaHalls::Repository.find(params[:id])`

Reply:
So, I don't need use_case index? at all?

Reply:
That's right. In fact `CinemaHalls::UseCases::Show.new.call(id: params[:id])` just call `CinemaHalls::Repository.new.find(params[:id])`, nothing more. The same with Index use_case.

Reply:
Now the problem is that POST test seance and POST test reservation say 422.